webapp/crashstats/crashstats/tests/test_views.py

```diff
@@ -475,15 +476,30 @@ def test_report_index(self, client, db, storage_helper, user_helper):
         # shows up.
         assert "The address where the crashing thread crashed." in content
 
-        metrics_mock.assert_timing(
-            "webapp.view.pageview",
-            tags=[
+        if socorro_settings.CLOUD_PROVIDER == "AWS":
+            metrics_mock.assert_timing(
+                "webapp.view.pageview",
+                tags=[
+                    "ajax:false",
+                    "api:false",
+                    "path:/report/index/_crashid_crash_id_",
+                    "status:200",
+                ],
+            )
+        else:
+            records = metrics_mock.filter_records(
```

Comment:
So the problem here that makes this wonky looking is that a `host` tag gets added and it's hard to know what the value would be. Markus doesn't have a good way of dealing with this. I wrote this up:

https://github.com/willkg/markus/issues/141

I might have been able to implement something here, but Markus tries hard to allow for unstable orders and does this:

https://github.com/willkg/markus/blob/8dcc2fad9178dc03853e666c1a84e90a30411150/src/markus/testing.py#L109-L110

In the meantime, I did this ridiculous thing. We can change it later when Markus grows an `AnyValue` thing.